src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -83,16 +142,53 @@ export const gravityStitchingEnvironment = (
           before: String
         ): UserAddressConnection
       }
+
+      extend type Partner {
+        searchCriteriaConnection(
+          first: Int,
+          last: Int,
+          before: String,
+          after: String,
+          artistID: ID,
+          highQuality: Boolean,
+          partnerID: String,
+          previewByArtist: Boolean
+          since: Int
+        ): SearchCriteriaConnection
+
+        viewingRoomsConnection(first: Int, after: String, statuses: [ViewingRoomStatusEnum!]): ViewingRoomsConnection
+      }
+
+      extend type Query {
+        curatedMarketingCollections(size: Int): [MarketingCollection]
+      }
+
       extend type SearchCriteria {
+        artistsConnection(
+          first: Int,
+          last: Int,
+          before: String,
+          after: String,
+          ids: [String],
+          sort: ArtistSorts,
+          term: String
+        ): ArtistConnection
```

Comment:
Another addition - only this time things are inverted. Here we're connecting `artistsConnection` (from local Metaphysics types) to a _Gravity_ type.